src/main/java/com/trimetrica/operationsandciclicfunctions/PrintNumberService.java

```diff
@@ -0,0 +1,32 @@
+package com.trimetrica.operationsandciclicfunctions;
+
+public class PrintNumberService {
+    public static void main(String[] args) {
+        int a = 500, limit = 500;
+
+        while (a > 0) {
+            System.out.print(a + ", ");
```

Comment:
You can make postincrementation directly in print, so you can execute while condition without brackets with a single line